zebra-consensus/src/transaction.rs

```diff
@@ -1029,11 +1031,13 @@ where
             // It calls [`pallas::Affine::from_bytes`] to parse R and
```

Comment:
Nitpick: The comment above this line still references `redpallas`.

Can you search for any remaining references and update them?

Reply:
Good catch, fixed it and others in 5db57917c